install.sh

```diff
@@ -81,6 +88,30 @@ install_nvm_from_git() {
   return
 }
 
+#
+# Automatically install Node.js
+#
+nvm_install_node() {
+  local NODE_VERSION
+  NODE_VERSION="$(nvm_node_version)"
+
+  if [ -z "$NODE_VERSION" ]; then
+    echo "=> You can now install Node.js by running \`nvm install\`"
+    return 0
+  fi
+
+  echo "=> Installing Node.js version $NODE_VERSION"
+  nvm install $NODE_VERSION
+  local CURRENT_NVM_NODE
+
+  CURRENT_NVM_NODE="$(nvm_version current)"
+  if [ "$(nvm_version \"$NODE_VERSION\")" == "$CURRENT_NVM_NODE" ]; then
```

Comment:
i think the inner quotes here don't need to be escaped - double quotes can be nested